File: fizzbuzz.cpp
#include <iostream>

int main()
{
	for(unsigned int i = 1; i < 101; ++i)
	{
		cout << (i % 3 == 0 && i % 5 == 0 ? "FizzBuzz" : i % 3 == 0 ? "Fizz" : i % 5 == 0 ? "Buzz" : std::to_string(i)) << endl;
	}
	
	return 0;
}

File: fizzbuzz.go
package main

import(
    "fmt"
    "strconv"
)

func fizz_buzz(n int) string {
  if n % 3 == 0 && n % 5 == 0 {
    return "FizzBuzz"
  } else if n % 3 == 0 {
    return "Fizz"
  } else if n % 5 == 0 {
    return "Buzz"
  } else {
    return strconv.Itoa(n)
  }
}

func main() {
  for i := 1; i < 101; i++ {
    fmt.Println(fizz_buzz(i))
  }
}
